ext/rugged/rugged.c

```diff
@@ -368,6 +368,29 @@ static VALUE rb_git_cache_usage(VALUE self)
 	return rb_ary_new3(2, LL2NUM(used), LL2NUM(max));
 }
 
+/*
+ *  call-seq:
+ *    Rugged.signature_from_buffer(buffer[, encoding_name]) -> signature
+ *
+ * Parse the signature from the given buffer. If an encoding is given, the
+ * strings will be tagged with that encoding.
+ *
+ *    commit.author #=> {:email=>"tanoku@gmail.com", :time=>Tue Jan 24 05:42:45 UTC 2012, :name=>"Vicent Mart\303\255"}
```

Comment:
This is why my inbox is full of interesting deals from Nigerian Princes. Oh well, guess that's what I get for re-implementing Git.